styles/components/BlockImageText.scss

```diff
@@ -1,7 +1,10 @@
 .BlockImageText {
+  &__description.small > div > span > p {
+    line-height: 1.5rem;
+  }
+
   &__description > div > span > p {
     padding-bottom: 2rem;
-    line-height: 1.5rem;
```

Comment:
I made this change since I found this bug(screenshot below) from /capabilities page. It was from this PR's `line-height` change. Fixed preview is [here](https://sanctu-dot-41ncm6i6o-sanctucompu.vercel.app/capabilities).

<img width="900" alt="Screen Shot 2023-02-10 at 11 01 37 AM" src="https://user-images.githubusercontent.com/12539032/218141162-74653543-e166-46f1-b59a-5fe86ed70cb0.png">